### PR TITLE
extend lastcmd mode to support all the previous cmds

### PR DESCRIPTION
**Pull request by tw4452852**
Sometimes, I found that what I want is not just last command, but even older one.

### CODE REVIEW

**Comment by xiaq** [comment]
I feel "rolling over" to the previous/next command is not exactly idea:

- I might hold down Up or Down to navigate to a far away item, and the worst thing that would happen is that I would stop at the first or last item. But now there's a danger of going over to another command
- It can also take a bit too many keystrokes to fully go over one command

Left and Right are not used in lastcmd mode, what about using them to navigate commands?

**Comment by tw4452852** [comment]
@xiaq How about using `Alt + <Up/Down>`? I prefer to reserve `<Left/Right>` to be used as selecting left/right item in the future.

**Comment by xiaq** [comment]
> @xiaq How about using `Alt + <Up/Down>`? I prefer to reserve `<Left/Right>` to be used as selecting left/right item in the future.

Alt-Up/Down sound fine too!

**Comment by tw4452852** [comment]
> > @xiaq How about using `Alt + <Up/Down>`? I prefer to reserve `<Left/Right>` to be used as selecting left/right item in the future.
> 
> Alt-Up/Down sound fine too!

Updated in the latest version.

File: pkg/cli/tk/listbox.go [hunk 1]
```diff
@@ -284,6 +294,12 @@ func (w *listBox) Handle(event term.Event) bool {
 	case term.K(ui.Enter):
 		w.Accept()
 		return true
+	case term.K(ui.Up, ui.Alt):
+		w.OnUnderFlow(w)
+		return true
+	case term.K(ui.Down, ui.Alt):
+		w.OnOverFlow(w)
+		return true
```
Comment (xiaq) [inline pkg/cli/tk/listbox.go:302]: It would be nicer if this is part of the keybinding rather than hardcoded in the event handler.

**Pull request merged**